### ISSUE TITLE
De-obfuscate ids on html page

### ISSUE THREAD

**Issue by dcolligan** [ga4gh/server#585]
Currently, with base64-obfuscated IDs, our html page looks like this:
```
Data

ReferenceSets

DataSets

ZGF0YXNldDE=

VariantSets

ZGF0YXNldDE6MWtnLXBoYXNlMQ==
ZGF0YXNldDE6MWtnLXBoYXNlMw==
ZGF0YXNldDE6dW1k
ReadGroupSets

ZGF0YXNldDE6cmVtb3Rl	
ZGF0YXNldDE6cmVtb3RlOndnRW5jb2RlVXdSZXBsaVNlcUJnMDJlc0cxYkFsblJlcDFfc2FtcGxl
ZGF0YXNldDE6ZGFudms=	
ZGF0YXNldDE6ZGFudms6Y2hyMTcuMS0yNTA=
ZGF0YXNldDE6bG93LWNvdmVyYWdl	
ZGF0YXNldDE6bG93LWNvdmVyYWdlOkhHMDAwOTYubWFwcGVkLklMTFVNSU5BLmJ3YS5HQlIubG93X2NvdmVyYWdlLjIwMTIwNTIy
ZGF0YXNldDE6bG93LWNvdmVyYWdlOkhHMDA1MzMubWFwcGVkLklMTFVNSU5BLmJ3YS5DSFMubG93X2NvdmVyYWdlLjIwMTIwNTIy
ZGF0YXNldDE6bG93LWNvdmVyYWdlOkhHMDA1MzQubWFwcGVkLklMTFVNSU5BLmJ3YS5DSFMubG93X2NvdmVyYWdlLjIwMTIwNTIy
```
I get that we don't want people relying on an ad-hoc convention, but perhaps there is a way to make this page more useful as a sanity check?  For instance, displaying the de-obfuscated ids alongside the obfuscated ones?  Thoughts?

**Comment by jeromekelleher** [comment]
How about we use the names as well to make it more clear? So, everywhere we have an ID now, we'd put a pair (name, id) to make it more readable.

**Comment by dcolligan** [comment]
Fixed in #586
